cli/Cargo.toml

```diff
@@ -32,9 +32,9 @@ tokio = { workspace = true, features = ["io-util"] }
 [package.metadata.winres]
 # This section defines the metadata that appears in the deno.exe PE header.
 OriginalFilename = "zinnia.exe"
-LegalCopyright = "© Protocol Labs & Zinnia contributors. Permissive License Stack (Apache-2.0 or MIT)."
-ProductName = "Filecoin Station"
-FileDescription = "Zinnia: Secure sandboxed runtime for Filecoin Station modules."
+LegalCopyright = "© Space Meridian & Zinnia contributors. Permissive License Stack (Apache-2.0 or MIT)."
+ProductName = "Checker Network"
+FileDescription = "Zinnia: Secure sandboxed runtime for Checker Network checker nodes"
```

Comment:
Let's call these Checker Network nodes, Checker Network checker nodes is redundant and doesn't sound good to me

Reply:
I share the same concerns.

FWIW, I used the term "checker nodes" because we have other kinds of nodes too - the nodes in the backend/orchestration layer.

I am going to use `runtime for Checker Network nodes`, we can easily change this later as needed.

Reply:
See af0030e